containers/allan_dip_bot/run_bot.py

```diff
@@ -260,6 +264,14 @@ async def play(
     parser.add_argument(
         "--outdir", type=str, help="output directory for game json to be stored"
     )
+    parser.add_argument(
+        "--outdir", type=str, help="output directory for game json to be stored"
+    )
+    parser.add_argument(
+        "--aggressiveness",
+        type=str,
+        help='aggressiveness of the bot ("A" - Aggressive, "M" - Moderate, "F" - Friendly)',
+    )
```

Comment:
The choices and default can be set automatically through `argparse`:

```suggestion
    parser.add_argument(
        "--aggressiveness",
        type=str,
        choices=["A", "M", "F"],
        default="M"
        help='aggressiveness of the bot ("A" - Aggressive, "M" - Moderate, "F" - Friendly)',
    )
```